src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import jwt from 'jsonwebtoken';
+
+export const authmiddleware = (req, res, next) => {
+  const auth = req.headers.authorisation;
```

Comment:
There is a typo in the header name 'authorisation'. It should be 'authorization' to correctly access the Authorization header in the request.